rockspec/apisix-master-0.rockspec

```diff
@@ -20,8 +20,8 @@ version = "master-0"
 supported_platforms = {"linux", "macosx"}
 
 source = {
-    url = "git://github.com/apache/apisix",
-    branch = "master",
+    url = "git://github.com/tokers/apisix",
+    branch = "improve/cmd_refactoring",
```

Comment:
why we need to change this?

Reply:
The `linux_apisix_master_luarocks_runner.sh` always fetches the master branch of APISIX which doesn't contains the `apisix/cmd` directory and causes the failure of CI, it's just a temporary change and will be reset before merging into master.

Reply:
which CI will fail? I think we need to resolve this first

Reply:
The `apisix_cli_test.sh`.